app/controllers/profile_preview_cards_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProfilePreviewCardsController < ApplicationController
```

Comment:
I renamed the controller and paths to follow the rails convention of having controllers with plural names